packages/fx-core/src/plugins/resource/frontend/configs.ts

```diff
@@ -79,21 +74,38 @@ export class FrontendConfig {
     }
 
     return new FrontendConfig(
-      appName,
       subscriptionId,
       resourceGroupName,
       location,
-      resourceNameSuffix,
       storageName,
       credentials
     );
   }
 
+  public syncToPluginContext(ctx: PluginContext): void {
+    Object.entries(this)
+      .filter((kv) => FrontendConfig.persistentConfigList.find((x: string) => x === kv[0]))
+      .forEach((kv) => {
+        if (kv[1]) {
+          FrontendConfig.setConfigIfNotExists(ctx, kv[0], kv[1]);
+        }
+      });
+  }
+
+  private static persistentConfigList = Object.entries(FrontendConfigInfo).map((kv) => kv[1]);
```

Comment:
Nit: could be `Object.values(FrontendConfigInfo)`.

Reply:
Updated.